src/com/kainos/tdp/employeemanager/Employee.java

```diff
@@ -0,0 +1,78 @@
+package com.kainos.tdp.employeemanager;
+
+public class Employee {
+	
+	private String employeeNumber, name, address, nationalInsuranceNo, IBAN;
+	private float startingSalary;
+	
+	
+	public Employee(String _employeeNumber, String _name, String _address, String _nationalInsuranceNo, String _IBAN, float _startingSalary) {
```

Comment:
```suggestion
	public Employee(String employeeNumber, String name, String address, String nationalInsuranceNo, String IBAN, float startingSalary) {
```
... fix usages below